pysatSpaceWeather/instruments/sw_apo.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+"""Supports apo index values.
+
+Properties
+----------
+platform
+    'sw'
+name
+    'apo'
+tag
+    - 'now' Near real-time nowcast apo data from GFZ
+inst_id
+    '30min', '60min'
```

Comment:
There's a mixed style throughout this package in the top-level docstring.  Sometimes inst_id uses bullets, sometimes not.  tag generally does.  Using bullets would allow a bit more description, which we don't get when defining in the code.

Reply:
I like the bullets idea, will apply it.